STT/Dockerfile.train

```diff
@@ -0,0 +1,249 @@
+FROM nvcr.io/nvidia/tensorflow:22.02-tf1-py3
+
+ARG stt_repo=coqui-ai/STT
+ARG stt_branch=fcec06bdd89f6ae68e2599495e8471da5e5ba45e
+ARG stt_sha1=fcec06bdd89f6ae68e2599495e8471da5e5ba45e
+ARG cc_repo=mozilla/CorporaCreator
+ARG cc_sha1=73622cf8399f8e634aee2f0e76dacc879226e3ac
+ARG kenlm_repo=kpu/kenlm
+ARG kenlm_branch=87e85e66c99ceff1fab2500a7c60c01da7315eec
+
+# Model parameters
+ARG model_language=fr
+ENV MODEL_LANGUAGE=$model_language
+
+# Training hyper-parameters
+ARG train_batch_size=64
+ENV TRAIN_BATCH_SIZE=$train_batch_size
+
+ARG dev_batch_size=64
+ENV DEV_BATCH_SIZE=$dev_batch_size
+
+ARG test_batch_size=64
+ENV TEST_BATCH_SIZE=$test_batch_size
+
+ARG n_hidden=2048
+ENV N_HIDDEN=$n_hidden
+
+ARG epochs=40
+ENV EPOCHS=$epochs
+
+ARG learning_rate=0.0001
+ENV LEARNING_RATE=$learning_rate
+
+ARG dropout=0.3
+ENV DROPOUT=$dropout
+
+ARG lm_top_k=500000
+ENV LM_TOP_K=500000
+
+ARG lm_alpha=0.0
+ENV LM_ALPHA=$lm_alpha
+
+ARG lm_beta=0.0
+ENV LM_BETA=$lm_beta
+
+ARG beam_width=500
+ENV BEAM_WIDTH=$beam_width
+
+ARG early_stop=1
+ENV EARLY_STOP=$early_stop
+
+ARG amp=0
+ENV AMP=$amp
+
+# Skipping batch test to avoid hanging processes
+# Should be set to 0 by default once STT#2195 is fixed
+# See https://github.com/coqui-ai/STT/issues/2195 for more details
```

Comment:
still a thing?

Reply:
Yep, I made a workaround by adding the `--skip_batch_test` flag but the hang still happens.

Reply:
https://github.com/coqui-ai/STT/pull/2205